src/bbs2gh/Commands/MigrateRepo/MigrateRepoCommandHandler.cs

```diff
@@ -198,6 +200,22 @@ private async Task<string> UploadArchiveToAws(string bucketName, string archiveP
         return archiveBlobUrl;
     }
 
+    private async Task<string> UploadArchiveToGithub(string org, string archivePath)
+    {
+#pragma warning disable IDE0063
```

Comment:
Is there a reason the warning was disabled? I think we can use `using declaration` here.